includes/class-wc-calypso-bridge-ecommerce-admin-menu.php

```diff
@@ -644,4 +656,14 @@ public function add_jetpack_menu() {
 			} );
 		}
 	}
+
+	/**
+	 * Check if the site has Entrepreneur plan
+	 *
+	 * @return bool
+	 */
+	protected function has_entrepreneur_plan() {
+		$current_plan = get_option( 'jetpack_active_plan',  array() );
+		return ! empty( $current_plan ) && 'ecommerce-bundle' === $current_plan['product_slug'];
```

Comment:
This will work for the "full" Entrepreneur plan, however, the Entrepreneur Trial plan currently has the follwing `product_slug`: `ecommerce-trial-bundle-monthly`.

I think this `product_slug` shouldn't change when launching. Maybe @andregardi could confirm. 🙂

Reply:
Correct, we are not changing the product slug. Good catch @ivan-ottinger.